### PR TITLE
[16.0][MIG] partner_address_two_lines: Migration to 16.0

### PR DESCRIPTION
**Pull request by yankinmax**
Depends on:
- [x] https://github.com/OCA/partner-contact/pull/1471

ref: RD-622

### CODE REVIEW

**Comment by yankinmax** [comment]
@pedrobaeza this test fails, because it violates postrgres constrain of duplicate state in Switzerland.
https://github.com/OCA/partner-contact/actions/runs/4201638823/jobs/7288843103#step:8:297
But, actually state with such code already exists in Odoo:
https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/data/res.country.state.csv#L1449

**Comment by yankinmax** [comment]
@pedrobaeza ping

**Review by damdam-s — APPROVED** [review]

File: partner_address_two_lines/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Partner address in two lines",
+    "version": "16.0.1.0.2",
```
Comment (simahawk) [inline partner_address_two_lines/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by simahawk — COMMENTED** [review]
LG but you should fix the version

**Review by simahawk — APPROVED** [review]

**Comment by yankinmax** [comment]
@simahawk can you pls help to get it merged? THX

**Pull request merged**